ui/app/utils/properties/watch.js

```diff
@@ -24,6 +24,9 @@ export function watchRecord(modelName) {
           wait(throttle),
         ]);
       } catch (e) {
+        if (controller && action) {
+          controller.send(action);
+        }
```

Comment:
This is not an approach I love, but it at least displays the error! Another possibility is to pass a callback function when `perform`ing the task.

Or the error could be attached to the Ember Data model directly!

Reply:
ALSO this currently collapses all kinds of errors into one interpretation, that the model has been deleted. It probably needs to be more targeted.